charts/rancher-istio/1.4.800/questions.yaml

```diff
@@ -1,3 +1,3 @@
 labels:
-  rancher.istio.v1.4.700: 1.4.7
+  rancher.istio.v1.4.800: 1.4.8
 rancher_min_version: 2.3.4-rc1
```

Comment:
Need to ask if we should also deprecate the 1.4.700 chart, think the answer is no.

Reply:
per Denise we will be deprecating, so I will update the pr